ark/phenotyping/preprocess.py

```diff
@@ -76,16 +78,16 @@ def create_pixel_matrix(img_xr, seg_labels, fovs=None, channels=None, blur_facto
 
         # turn into frequency
         pixel_mat.loc[:, channels] = pixel_mat.loc[:, channels].div(
-            pixel_mat.sum(axis=1), axis=0)
-
-        # 99.9% normalization
-        pixel_mat.loc[:, channels] = pixel_mat.loc[:, channels].div(
-            pixel_mat.quantile(q=0.999, axis=1), axis=0)
+            pixel_mat.loc[:, channels].sum(axis=1), axis=0)
```

Comment:
Adding a slightly more descriptive comment here would be useful. i.e. normalize each row by total marker counts to convert raw counts to frequencies